test_lib/compaction.py

```diff
@@ -4,9 +4,28 @@
 
 import yaml
 
+from sdcm.cluster import BaseNode
+
 LOGGER = logging.getLogger(__name__)
 
 
+class GcMode(Enum):
+    REPAIR = "repair"
+    DISABLED = "disabled"
+    TIMEOUT = "timeout"
+    IMMEDIATE = "immediate"
+
+    # TODO: uncomment and use this method when 5.0 query-table-extensions issue is fixed:
```

Comment:
anyway I don't see it as required.
Still we'll be able to create `GcMode` from string like this: `GcMode("repair")`. Just matching enum values to values returned from scylla will be enough.

Reply:
Very well. i think this matching had some issues on previous python versions.. seems to be working better now. removed.